### DISCUSSION TITLE
Protection DDOS

### DISCUSSION THREAD

**Discussion by vMax-dev1** [pelican-dev/panel#151] [Ideas]
A system like moonlight, which protects the flooding of TCP packets by banning ips for a configurable duration on all nodes when they reach a certain limit

So it would only protect the nodes but not the panel

**Comment by Sharktheone** [comment]
That's most likely more a job from a firewall than from the pelican. I guess, the easiest you can do is to set up crowdsec and let it look into server/docker logs. I don't really think it's necessary to have a direct integration for a firewall or a service like crowdsec.

**Comment by lancepioch** [comment]
Sorry, this is way beyond the scope of this project!
